src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -259,8 +271,10 @@ public void render(Graphics g)
     // Draw the currently-acting unit so it's on top of everything.
     if( null != currentActor )
     {
-      unitArtist.drawUnit(mapGraphics, currentActor, actorCoord.xCoord, actorCoord.yCoord, fastAnimIndex);
-      unitArtist.drawUnitIcons(mapGraphics, currentActor, actorCoord.xCoord, actorCoord.yCoord, animIndex);
+      SpriteLibrary.getMapUnitSpriteSet(currentActor)
```

Comment:
Why are we bypassing `UnitArtist` now?
Also this won't draw unit icons when a unit is selected anymore.

Reply:
Didn't want to change `UnitArtist` to take draw-space coordinates. (Tried it, and it was a little ugly having an overload that take the same basic info of two functions)

Second point is an intended change, though I guess I could revert that.